server/db/models/story.js

```diff
@@ -0,0 +1,31 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Story = db.define('story', {  //Story belongs to a story, will have story id via association
+  name: {
+    type: Sequelize.STRING,
+    validate: {
+      notEmpty: true
+    }
+  },
+  url: {
+    type: Sequelize.STRING,
+    validate: {
+      notEmpty: true
+    }
+  },
+  genre: { //helpful for filtering in public browse setting
+    type: Sequelize.STRING,
+    validate: {
+      notEmpty: true
+    }
+  },
+  approxLength: {  //helpful for filtering in public browse setting
+    type: Sequelize.INTEGER
+  },
+  releaseDate: {
```

Comment:
look into date (for postgres) test it! to validate storage of both date and time